pallets/staking/src/lib.rs

```diff
@@ -282,6 +282,9 @@ pub mod pallet {
 		/// Signer is not an owner of the staking position.
 		Forbidden,
 
+		/// Remove vote is not allowed when referendum is finished and staking position exists.
+		RemoveVoteNotAllowed,
```

Comment:
name is too generic, just call it `StakingPositionsExists`

Reply:
that's little bit confusing imo. there is already similar error that staking position exists but used in different context.

i believe it should explicitly state that remove vote is not allowed.